<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>682</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="student">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>901</width>
      <height>591</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="addStudent">
     <attribute name="title">
      <string> AddStudent</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add New Student</string>
      </property>
     </widget>
     <widget class="QLabel" name="studentNameLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Student Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="fatherNameLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Father's Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="regnLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Registration No:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="studentName">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>90</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="regno">
      <property name="geometry">
       <rect>
        <x>162</x>
        <y>153</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="fatherName">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>220</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="marksLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Marks:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>360</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>DSA:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>410</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>IWP:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>460</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Networks:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dsaMarks">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>360</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="iwpMarks">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>410</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="networksMarks">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>460</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="submitStudent">
      <property name="geometry">
       <rect>
        <x>379</x>
        <y>404</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="studentDetails">
     <attribute name="title">
      <string>StudentDetails</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>321</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Enter a registration number to search for:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="searchRegno">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>60</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="searchStudent">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>60</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Student Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="student_name">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>150</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Registration no.</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="reg_no">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>210</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Marks:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>360</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>DSA</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>420</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>IWP</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>480</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Networks</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dsa_marks">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>353</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="iwp_marks">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>413</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="networks_marks">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>473</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="father_name">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>270</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Father's Name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteStudent">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>480</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete student</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="EditStudent">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <attribute name="title">
      <string>EditStudent</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>881</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>12</number>
       </property>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Enter registration number to search for:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="regno_edit"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>601</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>77</number>
       </property>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="name_edit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>601</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Father's Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fname_edit"/>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Marks:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>450</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>370</y>
        <width>91</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>DSA</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>IWP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Networks</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>370</y>
        <width>201</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="dsa_edit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="iwp_edit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="networks_edit"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="edit_btn">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>410</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Changes</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="notepad">
     <attribute name="title">
      <string>Notepad</string>
     </attribute>
     <widget class="QPushButton" name="notepadButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>190</y>
        <width>221</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Click to open Notepad</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Calculator">
     <attribute name="title">
      <string>Calculator</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>271</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Click to open calculator</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>937</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
